modules/con-vuln-sources.adoc

```diff
@@ -36,8 +36,10 @@ link:https://security.access.redhat.com/data/csaf/v2/vex/[Red{nbsp}Hat VEX]:: Us
 +
 {product-title-short} might list a different number of vulnerabilities when you are scanning with a {product-title-short} version that uses OVAL, such as {product-title-short} version 4.5, and a version that uses VEX, such as version 4.6. For example, {product-title-short} no longer displays vulnerabilities with a status of "under investigation," while these vulnerabilities were included with previous versions that used OVAL data.
 +
+For containers from the Red Hat Ecosystem Catalog, {product-title-short} provides an option to show only vulnerabilities from VEX data. VEX data for Red Hat images is the most accurate because vulnerabilities in those images have been vetted by the Red Hat security team and the results reported in VEX. Other vulnerability sources such as OSV can report vulnerabilities that Red Hat has determined are not applicable to the image. This can cause false positives in vulnerability results. The option to use VEX data for Red Hat containers is disabled by default. To enable this option, in Scanner V4 Matcher, set the environment variable `ROX_SCANNER_V4_RED_HAT_LAYERS_RED_HAT_VULNS_ONLY` to `true`. Be aware that in rare instances, using this option can cause valid vulnerabilities to not appear in scan results. For example, Red Hat does not track vulnerabilities for products that have reached end of life.
```

Comment:
Not sure if we want to call out Middleware, too - https://access.redhat.com/security/middleware_security_scanning_problem